source/postprocess/heat_flux_densities.cc

```diff
@@ -21,6 +21,7 @@
 
 #include <aspect/postprocess/heat_flux_densities.h>
 #include <aspect/utilities.h>
+#include <aspect/geometry_model/interface.h>
```

Comment:
why do you add this here? (and in the other files?)

Reply:
I had to remove geometry_model/interface.h from utilities.h to get it to compile.
Those files needed geometry_model/interface.h so I re-added it.